ecs.tf

```diff
@@ -10,15 +10,34 @@ data "template_file" "ecs_user_data" {
   }
 }
 
+data "aws_ami" "amazon_ecs_ami" {
+  most_recent = true
+
+  filter {
+    name   = "owner-alias"
+    values = ["amazon"]
+  }
+
+  name_regex = ".+-amazon-ecs-optimized$"
+
+  most_recent = true
+}
```

Comment:
We should either move the ami changes into another pull request, or at least split it out into another commit. If we do keep it in this pull request, the title needs changing.

Reply:
Done